en/skills/11. Tables.ipynb

```diff
@@ -0,0 +1,458 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "a4a92198",
+      "metadata": {
+        "id": "a4a92198"
+      },
+      "source": [
+        "# Working with Tables\n",
+        "\n",
+        "Tables are a simple way to organize and store data. They are commonly used in spreadsheets (think Excel or Google Sheets) and databases, where each row holds a record, and each column holds a different attribute of that record.\n",
+        "\n",
+        "CSV files (Comma-Separated Values) are one of the most popular formats for storing tables. This files can be easily exported from an Excel or Google Sheets file (or opened via one of these applications)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Exporting a CSV"
+      ],
+      "metadata": {
+        "id": "LtqgsVRMfLvL"
+      },
+      "id": "LtqgsVRMfLvL"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Exporting from Excel"
+      ],
+      "metadata": {
+        "id": "OXBo4HBy6w9k"
+      },
+      "id": "OXBo4HBy6w9k"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "\n",
+        "- Open your Excel file.\n",
+        "- Click on File in the top-left corner.\n",
+        "- Select Save As.\n",
+        "- Choose a location to save the file.\n",
+        "- In the Save as type dropdown, select CSV (Comma delimited) (*.csv).\n",
+        "- Click Save.\n",
+        "\n",
+        "Note:\n",
+        "- A warning may pop up, stating that some features of the workbook may be lost if saved in CSV format. Click Yes to confirm.\n"
+      ],
+      "metadata": {
+        "id": "KI3_vdMe6n9n"
+      },
+      "id": "KI3_vdMe6n9n"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Exporting from Google Sheets"
+      ],
+      "metadata": {
+        "id": "LSpApMWZ6yL_"
+      },
+      "id": "LSpApMWZ6yL_"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "- Open your Google Sheets file.\n",
+        "- Go to File in the menu bar.\n",
+        "- Hover over Download.\n",
+        "- Select Comma-separated values (.csv).\n",
+        "- The file will automatically download to your default download location."
+      ],
+      "metadata": {
+        "id": "HQP1W-s060ya"
+      },
+      "id": "HQP1W-s060ya"
+    },
+    {
+      "cell_type": "markdown",
+      "id": "fec8ab9f",
+      "metadata": {
+        "id": "fec8ab9f"
+      },
+      "source": [
+        "## Reading and Writing CSV Files\n",
+        "Just like reading from a CSV file, you can also write new data into a CSV file using Python’s `csv` module.\n",
+        "\n",
+        "#### Example (Writing):"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "e491bbfc",
+      "metadata": {
+        "id": "e491bbfc"
+      },
+      "outputs": [],
+      "source": [
+        "import csv\n",
```

Comment:
What about using pandas? I think it's pretty common and makes some things a bit easier. But if there's a good reason to use `csv` then that totally works

Reply:
Same a suggestion. The idea is to have pandas on a different skill I believe.